### PR TITLE
Adds the Pneumatic Cannon

### LINKED ISSUE #5399: Pneumatic Cannon!
## Description

The pneumatic cannon is a unique gas-powered gun that can fire anything that is loaded into it. Items fired from the cannon is identical in function to throwing an item, but deal increased throwing damage and have a greater range. Pneumatic cannons have a variable gas valve with three settings for adjustable firepower at the cost of greater gas consumption.

The pneumatic cannon is the most vital piece in the arsenal of comedy. A chef is nothing without a backpack full of pneumatic cannons loaded with copy pastas. It would be incredible to get this legendary firearm in the game as soon as possible.

traits of the pneumatic cannon that would be nice to have but are not _required_ for a first implementation:

- shooting people's mouths feeds an item to them if they don't have a face covering
- the maximum power setting knocks down the wielder when fired.
- pies should pie people in the face

### PR DESCRIPTION
**Pull request by AtnerNT**
Closes #5399 

### Changelog:

CL: [New] Adds the Pneumatic Cannon and its ability to be researched, comes with three adjustable power settings, the highest power setting will knock the wielder down.
CL: [New] Food thrown at the mouth is now consumed.

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
# FORCE FEEDING WILL BE REAL IN UNITYSTATION IN 2013

**Comment by AtnerNT** [comment]
Ready for merge

File: UnityProject/Assets/Scripts/Items/Science/PneumaticCannon.cs [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+		SoundManager.PlayNetworkedAtPos(firingSound, transform.position, sourceObj: interaction.Performer);
+		if (pressureToDischarge >= (int)PressureSetting.High * CANNON_MAX_PRESSURE) interaction.PerformerPlayerScript.RegisterPlayer.ServerStun(dropItem: false);
+
+		//We assume a barrel area of 0.1 square meters applied for 1/10th of a second.
+		var physics = itemSlot.ItemObject.GetComponent<UniversalObjectPhysics>();
+		if (physics == null) return;
+
+		Inventory.ServerDrop(itemSlot);
+		physics.NewtonianNewtonPush(interaction.TargetVector, pressureToDischarge * 0.05f, 0.5f, 0.5f, interaction.TargetBodyPart, gameObject, 1f);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Science/PneumaticCannon.cs:255]: Technically you have to accounts for the pressure outside of the container if you want to be fully accurate, 
gameplay impact your Canon will be more powerful in space versus on the station

Comment (AtnerNT) [reply UnityProject/Assets/Scripts/Items/Science/PneumaticCannon.cs:255]: Would at most be a difference of 100kPa on a 500kPa (weakest setting) shot. Could add but seemed unnessacary

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Science/PneumaticCannon.cs:255]: ok all good

**Review by MaxIsJoe — APPROVED** [review]

**Pull request merged**